### PR TITLE
Chapter 3

### PR DESCRIPTION
**Pull request by alexVOLTS**

### CODE REVIEW

File: Chapter_3/2.c [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+int main(void)
+{
+    
+    char ascii;
+    printf("Enter ASCII code:\n");
+    scanf("%d", &ascii);
+
+    printf("ASCII code for %d is %c", ascii, ascii);
+    getchar();
+    getchar();
```
Comment (dimmykar) [inline Chapter_3/2.c:15]: Why does the function call twice? First time to stop program until the key pressed and what about the second?

File: Chapter_3/5.c [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#include <locale.h>
+#include <limits.h>
+#include <float.h>
+
+int main(void)
+{
+    int year;
+    printf ("Введите возраст в годах:\t");
+    scanf ("%d", &year);
+    int sec = year*3.156E7;
```
[the remark this answers is not in the document]
Comment (dimmykar) [reply Chapter_3/5.c:11]: Also use the define or const variable for magic number `3.156E7`

File: Chapter_3/6.c [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+#include <limits.h>
+#include <float.h>
+
+int main(void)
+{
+    float kvarta;
+    printf ("Введите объем воды в квартах:\t");
+    scanf ("%f", &kvarta);
+    float weight = 950;
+    float molecul = 3.0E-23;
```
Comment (dimmykar) [inline Chapter_3/6.c:12]: If this is constant value, then you need to use const variable

```c
const float molecul = 3.0E-23;
```

File: Chapter_3/7.c [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+int main(void)
+{
+    float growth;
+    printf ("Введите рост в дюймах: \t");
+    scanf ("%f", &growth);
+    float cm = growth*2.54;
+    printf ("\nВаш рост В дюймаx: %f",growth);
+    printf ("\nВаш рост В сантиметрах: %f", cm);
+    
+        if (cm > 180)
```
Comment (dimmykar) [inline Chapter_3/7.c:15]: Magic number `180`

**Review by dimmykar — CHANGES_REQUESTED** [review]
Good work! But you have some nitpicks

File: Chapter_3/7.c [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+#include <locale.h>
+#include <limits.h>
+#include <float.h>
+
+int main(void)
+{
+    float height;
+    printf("Input your height in inches: \t");
+    scanf("%f", &height);
+    float cm = height * 2.54;
```
Comment (dimmykar) [inline Chapter_3/7.c:11]: Magic number, use define, please

**Review by dimmykar — CHANGES_REQUESTED** [review]

File: Chapter_3/3.c [hunk 6]
```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+#include <locale.h>
+#include <limits.h>
+#include <float.h>
+
+int main(void)
+{
+    printf ("\a Напуганная внезапным звуком, Вика вскрикнула: \n \"Во имя всех звезд, что это было!\""); // \a - предупредительный звук (стандарт ANSI C)
```
[the remark this answers is not in the document]
Comment (alexVOLTS) [reply Chapter_3/3.c]: Yeah it is LOL

File: Chapter_3/2.c (hunk 1 above)
Comment (alexVOLTS) [reply Chapter_3/2.c]: Have no idea, found that form in the internet. Will fix this soon

File: Chapter_3/6.c [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+#include <locale.h>
+#include <limits.h>
+#include <float.h>
+
+#define WEIGHT     950.0f
+#define MOLECUL    3.0E-23f
```
Comment (dimmykar) [inline Chapter_3/6.c:7]: It will be `0.0000` for `float`. Use `3.0E-23` instead

File: Chapter_3/5.c [hunk 8]
```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <locale.h>
+#include <limits.h>
+#include <float.h>
+
+#define SECONDS    3.156E7f
```
Comment (dimmykar) [inline Chapter_3/5.c:6]: Do not use `float` for very small values. Use `3.156E7` instead (`double`)

**Pull request merged**